components/automate-ui/src/app/entities/automate-settings/automate-settings.requests.ts

```diff
@@ -22,49 +25,111 @@ export class AutomateSettingsRequests {
   // fetchJobSchedulerStatus sends an HTTP GET Request to read the status of the JobScheduler
   // living inside the ingest-service
   public fetchJobSchedulerStatus(_params): Observable<JobSchedulerStatus> {
-    const url = `${RETENTION_URL}/nodes/status`;
+    // NEW ENDPOINT FOR DATA-LIFECYCLE
+    const url = '/api/v0/data-lifecycle/status';
 
     return this.http
       .get<RespJobSchedulerStatus>(url).pipe(
       map((res) => this.convertResponseToJobSchedulerStatus(res)));
   }
 
-  // configureIngestJob sends an HTTP POST Request to the provided ingest job to configure
+  // configureIngestJob sends an HTTP PUT Request to the provided ingest job to configure
   // it with the provided threshold and running state
-  configureIngestJob(job: IngestJob): Observable<any> {
-    let url: string;
+  configureIngestJobs(jobs: IngestJob[]): Observable<any> {
+    const url = '/api/v0/data-lifecycle/config';
 
-    switch (job.name) {
-      case IngestJobs.MissingNodes: {
-        url = `${RETENTION_URL}/nodes/missing-nodes/config`;
-        break;
-      }
-      case IngestJobs.MissingNodesForDeletion: {
-        url = `${RETENTION_URL}/nodes/missing-nodes-deletion/config`;
-        break;
+    const body: JobRequestBody = {
+      infra : {
+        job_settings: [
+          {
+            name: 'periodic_purge_timeseries',
+            purge_policies: {
+              elasticsearch: []
+            }
+          }
+        ]
+      },
+      compliance: {
+        job_settings: [
+          {
+            name: 'periodic_purge',
+            purge_policies: {
+            elasticsearch: []
+            }
+          }
+        ]
+      },
+      event_feed: {
+        job_settings: [
+          {
+            name: 'periodic_purge',
+            purge_policies: {
+            elasticsearch: []
+            }
+          }
+        ]
       }
-      case IngestJobs.DeleteNodes: {
-        url = `${RETENTION_URL}/nodes/delete-nodes/config`;
-        break;
+    };
+
+    jobs.forEach((job: IngestJob): void => {
+      let thisJob;
+      let matchedNest;
```

Comment:
More strong types needed here to avoid "compiler forgiveness". Before you make this change, you could have a typo in `matchedNest.purge_policies` and the compiler would not complain. Once you strongly type these, it is much safer.
```suggestion
      let thisJob: UnfurledJob;
      let matchedNest: JobRequestComponent;
```